packages/vue/src/components/result/ResultList.jsx

```diff
@@ -0,0 +1,113 @@
+import { RLConnected } from './ReactiveList.jsx';
+import Title from '../../styles/Title';
+import VueTypes from 'vue-types';
+import ListItem, { container, Image } from '../../styles/ListItem';
+import types from '../../utils/vueTypes.js';
+import { helper } from '@appbaseio/reactivecore';
+
+const { getClassName } = helper;
+
+const ResultList = {
+	name: 'ResultList',
+	props: {
+		currentPage: VueTypes.number.def(0),
+		includeFields: types.includeFields.def(['*']),
+		// component props
+		className: types.string,
+		componentId: types.stringRequired,
+		dataField: types.stringRequired,
+		defaultQuery: types.func,
+		excludeFields: types.excludeFields.def([]),
+		innerClass: types.style,
+		listClass: VueTypes.string.def(''),
+		loader: types.title,
+		onAllData: types.func,
+		onData: types.func,
+		onResultStats: types.func,
+		onNoResults: VueTypes.string.def('No Results found.'),
+		pages: VueTypes.number.def(5),
+		pagination: VueTypes.bool.def(false),
+		paginationAt: types.paginationAt.def('bottom'),
+		react: types.react,
+		showResultStats: VueTypes.bool.def(true),
+		size: VueTypes.number.def(10),
+		sortBy: types.sortBy,
+		sortOptions: types.sortOptions,
+		stream: types.bool,
+		URLParams: VueTypes.bool.def(false),
+		target: VueTypes.string.def('_blank')
+	},
+	render() {
+		const { onData, ...props } = this.$props;
+		return (
+			<RLConnected
+				{...{
+					props: {
+						onData: this.renderAsList,
+						...props,
+						listClass: container,
+					}
+				}}
+			/>
+		);
+	},
+	methods: {
+		renderAsList({ item, triggerClickAnalytics }) {
+			const result = this.$props.onData(item);
```

Comment:
@jyash97 Also use scopedSlots here.

Reply:
Like a condition if `scopedSlot` is not defined we can use `$props` ?

Reply:
Yes, similar to reactive list.
```
const onData = this.$scopedSlots.onData || this.$props.onData;
```

Reply:
Got it, but I am not aware, how scopedslot would return object?